packages/dev/core/src/Engines/WebGL/webGLRenderTargetWrapper.ts

```diff
@@ -43,6 +45,38 @@ export class WebGLRenderTargetWrapper extends RenderTargetWrapper {
      */
     public _currentLOD = 0;
 
+    public override get depthStencilTexture() {
+        return this._depthStencilTexture;
+    }
+
+    public override set depthStencilTexture(texture: Nullable<InternalTexture>) {

```

Comment:
is the override keyword needed if the setter is not in the base class ?

Reply:
The setter (as well as the getter) is defined in the base class (`RenderTargetWrapper`).

Reply:
was looking locally...